8_about.md

```diff
@@ -9,7 +9,9 @@ toc: true
 
 ## Goal
 
-Thousands of lifesaving organs are wasted every year. Our goal is to highlight underlying causes that can be addressed. We investigated and analyzed the current organ donation system, looking specifically at common practices by OPOs, technology managed by government contractors, and the governance structure within the organ donation system. These reports outline what we found, and what recommendations we have for improving the system. 
+Thousands of lifesaving organs are wasted every year. Our goal is to highlight underlying causes that can be addressed. We investigated and analyzed the current organ donation system, looking specifically at common practices by OPOs, technology managed by government contractors, and the governance structure within the organ donation system. These reports outline what we found, and what recommendations we have for improving the system.
+
+The goal is to get more organs.
```

Comment:
Um is that really the goal? Wouldn't that mean more deaths? We should be explicit